Workaround for Backer and Sponsor Icon Sizes in Edge/IE

mattbierner opened the pull request:
## Bug

Edge/IE seems to have a bug rendering empty backer image icons:

![image](https://cloud.githubusercontent.com/assets/212305/19371605/54109adc-9169-11e6-9a5f-efbbb6f39253.png)

I don't know what is going wrong specifically, but believe it may have something to do with the svg file having a `viewbox` but not having a `width`.
## Fix

Add an explicit `max-height` rule in the css to fix this:

![image](https://cloud.githubusercontent.com/assets/212305/19371565/14a9d0de-9169-11e6-9d31-2d4067ce318c.png)

Tested on Edge and Chrome on windows to confirm everything renders correctly.

boneskull commented:
@mattbierner Thanks.  Can you please [sign the CLA](https://cla.js.foundation/mochajs/mochajs.github.io?pullRequest=56)?  You may want to rebase first so you've committed this under one GitHub account, because the CLA is linked to it.

mattbierner committed:
**Bug**
Edge renders the icon for a backer without an image to take up the entire screen.

**Fix**
Add an explicit max-height in css.

mattbierner commented:
Sorry for the delay getting back here. I've update the PR to be a single change from my non-work account and signed the CLA, although it is still listed as unsigned.

boneskull commented:
thanks!

boneskull commented:
@mattbierner see the "checks" section--the bot has your signature.  it's all good
